src/components/template/TheNavBar.vue

```diff
@@ -6,10 +6,10 @@
             <router-link @click.native="resetAlert" :to="{ name: 'UserSearch'}">Users</router-link>
         </li>
         <li id="event-log-link" :class="$route.name == 'EventLog' ? 'active' : 'inactive'">
-            <router-link @click.native="resetAlert" :to="{ name: 'EventLog'}">Event Log</router-link>
+            <router-link @click.native="resetAlert" :to="{ name: 'EventLog'}">Login Events</router-link>
         </li>
         <li id="admin-event-log-link" :class="$route.name == 'AdminEventLog' ? 'active' : 'inactive'">
-            <router-link :to="{ name: 'AdminEventLog'}">Admin Event Log</router-link>
+            <router-link @click.native="resetAlert" :to="{ name: 'AdminEventLog'}">Administrative Events</router-link>
```

Comment:
I think the names "Events" and "Admin Events" aren't clear enough. It sounds like the first should locally include the second: admin events are a kind of event.